pulsar-client-admin/src/main/java/com/yahoo/pulsar/client/admin/Brokers.java

```diff
@@ -67,4 +67,13 @@
      * @throws PulsarAdminException
      */
     Map<String, NamespaceOwnershipStatus> getOwnedNamespaces(String cluster, String brokerUrl) throws PulsarAdminException;
+    
+    /**
+     * Update value of any dynamic {@link ServiceConfiguration} 
+     * 
+     * @param key
+     * @param value
+     * @throws PulsarAdminException
+     */
+    void updateConfiguration(String configName, String configValue) throws PulsarAdminException;
```

Comment:
I have updated the document to call it out clearly and also renamed it as `updateDynamicConfiguration`. Do you think if we rename it to something else.?